canonical-web-and-design/ubuntu.com
# Remove mybrowser.fyi from issue-template

## Done

- Removes mybrowser.fyi from issue-template as this site no longer exists.
- Added the text 'Specify the browser(s) this issue effects.'

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Create a new issue against ubuntu.com, check the message doesn't include a link to mybrowser.fyi and has the new message mentioned above.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11217

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
